cng/hkdf.go

```diff
@@ -122,3 +122,8 @@ func ExpandHKDF(h func() hash.Hash, pseudorandomKey, info []byte, keyLength int)
 	}
 	return out, err
 }
+
+func bigUint32(b []byte) uint32 {
```

Comment:
Seems too easy to mix up with BigInt.

```suggestion
func bigEndianUint32(b []byte) uint32 {
```